jobs/component_jobs.groovy

```diff
@@ -57,23 +53,27 @@ repos.each { Map repo ->
             triggerPhrase('OK to test')
             orgWhitelist(['deis'])
             allowMembersOfWhitelistedOrgsAsAdmin()
-            // TODO: configure commit status messaging
-            // extensions {
-            //   commitStatus {
-            //     context('deploy to staging site')
-            //       triggeredStatus('starting deployment to staging site...')
-            //       startedStatus('deploying to staging site...')
-            //       statusUrl('http://mystatussite.com/prs')
-            //       completedStatus('SUCCESS', 'All is well')
-            //       completedStatus('FAILURE', 'Something went wrong. Investigate!')
-            //       completedStatus('PENDING', 'still in progress...')
-            //       completedStatus('ERROR', 'Something went really wrong. Investigate!')
-            //     }
-            //   }
+            extensions {
+              commitStatus {
+                context("Testing ${repo.name} PR")
+                triggeredStatus("After ${repo.name} build/deploy, kicking off tests via ${downstreamJobName}")
+                completedStatus('SUCCESS', 'All tests have passed!')
+                completedStatus('FAILURE', 'Build/deploy or tests have returned failure(s).')
+                completedStatus('ERROR', 'Something went wrong. Investigate!')
+              }
+            }
           }
         }
       }
 
+      publishers {
```

Comment:
Thoughts on removing these notifications?  All of the info should now be presented in the `workflow_test.groovy` notification seen in this PR (we now get the appropriate slack channel for that one)